includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -287,7 +278,9 @@ public function admin_scripts( $hook_suffix ) {
 			'is_oc_available'                       => WC_Stripe_Feature_Flags::is_oc_available(),
 			'oauth_nonce'                           => wp_create_nonce( 'wc_stripe_get_oauth_urls' ),
 			'is_sepa_tokens_enabled'                => 'yes' === $this->gateway->get_option( 'sepa_tokens_for_other_methods', 'no' ),
-			'has_other_bnpl_plugins'                => $has_other_bnpl_plugins_active,
+			'has_affirm_gateway_plugin'             => WC_Stripe_Helper::has_gateway_plugin_active( WC_Stripe_Helper::OFFICIAL_PLUGIN_ID_AFFIRM ),
+			'has_klarna_gateway_plugin'             => WC_Stripe_Helper::has_gateway_plugin_active( WC_Stripe_Helper::OFFICIAL_PLUGIN_ID_KLARNA ),
+			'has_other_bnpl_plugins'                => WC_Stripe_Helper::has_other_bnpl_plugins_active(),
```

Comment:
Another option here would be to reuse the return of both `WC_Stripe_Helper::has_gateway_plugin_active`. Not sure if it is worth it.